src/Bicep.Core/FileSystem/PathHelper.cs

```diff
@@ -12,7 +11,7 @@ public static class PathHelper
 
         private const string TemplateOutputExtension = ".json";
 
-        private const string BicepExtension = ".bicep";
+        private const string BicepExtension = LanguageConstants.LanguageFileExtension;
```

Comment:
@majastrz - because this is a private const field, I can remove it and replace it with `LanguageConstants.LanguageFileExtension` - semantics will be identical.